cmake/version.h.in

```diff
@@ -0,0 +1,29 @@
+#ifndef SKA_ASTROACCELERATE_VERSION_H
```

Comment:
Please could you adopt the naming convention for the header guards that are on this branch? This would be `ASTRO_ACCELERATE_VERSION_H_IN`.

Reply:
The header guard still looks unchanged to me.

Reply:
Whichever way I look at this file, it still looks like `SKA_ASTROACCELERATE_VERSION_H` to me. It should say `ASTRO_ACCELERATE_VERSION_H_IN`.